functionary/ui/templates/core/function_detail.html

```diff
@@ -60,14 +70,58 @@ <h1 class="title is-1">
             </div>
         {% endif %}
         {% if form is not None %}
-            <div class="field ml-4">
-                <label class="label">Create Task:</label>
-                <div class="ml-4">
-                    <form id="djangoForm"
-                          method="post"
-                          action="{% url 'ui:function-execute' %}">
-                        {% csrf_token %}
-                        <div class="has-background-white-ter">
+            {% if function.active %}
```

Comment:
Something definitely went wrong here because this template just fails to load.  You may want to just start with the template from main and re-apply your changes, rather than trying to sort out how to get it to rebase / merge properly.

Reply:
Started with main and reimplemented.